ocamlbuild/findlib.ml

```diff
@@ -153,7 +153,7 @@ let add_atom a l = match a, l with
   | A "", _ -> l
   | _ -> a :: l
 
-let compile_flags l =
+let include_flags ?(arg="-I") l =
```

Comment:
Why are you exposing an optional argument (the optional argument itself looks rather fishy to me) if it is not used in the patch. Do you have a need for it in an external plugin?

(If so, I think that rather than an obscure API, I would prefer a `locations : package list -> string list` function to be exposed that allows people to implement whatever variant of `include_flags` without hassle. Note that I think naming the function "include_flags" rather than "compile_flags" is a good change.)